app/components/file/report-btn/styles.scss

```diff
@@ -0,0 +1,93 @@
+.file-report-btn {
+  display: flex;
+  column-gap: 12px;
+
+  .btn-container {
+    column-gap: 8px;
+    position: relative;
+    border-radius: 4px;
+
+    .progress-loader {
+      position: absolute;
+      width: 0%;
+      height: 100%;
+      background-color: $primary;
+      bottom: 0;
+      left: 0;
+      opacity: 0.2;
+    }
+  }
+
+  .report-list-dropdown {
+    .dd-trigger {
+      height: inherit;
+      border-radius: 0.4em;
+      margin-bottom: 5px;
```

Comment:
Again, please don't use `px` units!!

Reply:
okay

Reply:
There is still lot of `px` units specified in that file even after your changes. Please fix those.